pages.de/common/head.md

```diff
@@ -0,0 +1,20 @@
+# head
+
+> Gibt den ersten Teil einer Datei aus.
+> Weitere Informationen: <https://www.gnu.org/software/coreutils/head>.
+
+- Gebe die ersten 'anzahl' Zeilen der Datei 'datei' aus:
```

Comment:
The non-English pages should be a translation from the main «English» pages. Could you please review these example descriptions? I understand that your first translation was made without translate the placeholders.

Reply:
So I should remove the placeholders from the translated example descriptions as there are no placeholders in the english example descriptions?

Reply:
I don't speak German, but I think something like this is better:
```suggestion
- Gebe die ersten anzahl Zeilen der Datei aus:
```

Reply:
Hmm, `anzahl` can be roughly translated by amount, so it makes sense only as a placeholder for e.g. a number.
I think it would be more appropriate to make the placeholders in the actual commands more descriptive and leave them out of the command's explanation.